css/style1.css

```diff
@@ -79,7 +79,7 @@ body{
     color:#3f7f5f
 }
 #copyright{
-    margin-top:0px;
+    margin-top:0;
```

Comment:
Please revert back this change.

Reply:
I am getting an issue there!

Reply:
What issue?
Can you please elaborate? is it CodeFactor checks failing?

Reply:
Yeah! CodeFactor checks are failing on reverting back the changes that I have made.

Reply:
Got it.
**Please check [this](https://stackoverflow.com/questions/4318471/property-0-or-property-0px-in-css)**

What do you say, should I ignore the checks or not?

Reply:
You may ignore the checks as that would not have any impact on the code.

Reply:
Cool:)
Thanks for the PR, I'll be merging it soon.

**You may find a [few](https://github.com/vinitshahdeo/HacktoberFest2K19/blob/master/README.md) more issues [here](https://vinitshahdeo.github.io/HacktoberFest2K19/)!**